packages/tupaia-web-server/src/routes/EntitiesRoute.ts

```diff
@@ -6,7 +6,7 @@
 import { Request } from 'express';
 import { Route } from '@tupaia/server-boilerplate';
 import camelcaseKeys from 'camelcase-keys';
-import { TupaiaWebEntitiesRequest } from '@tupaia/types';
+import { TupaiaWebEntitiesRequest, Entity } from '@tupaia/types';
```

Comment:
Are we using the `Entity` type here for anything?

Reply:
Oh good catch 👍